tests/basilisp/test_core_fns.lpy

```diff
@@ -2051,3 +2051,22 @@
       (eval '(ns test-core-fns.ns-with-import (:import [time :as time-alias])))
       (is (= *ns* (the-ns 'test-core-fns.ns-with-import)))
       (is (= "Thu Jan  1 00:00:01 1970" (eval '(time-alias/asctime (time-alias/gmtime 1))))))))
+
+
+;;;;;;;;;;
+;; load ;;
+;;;;;;;;;;
+
+(deftest load-test
```

Comment:
Can we just generate the `corpus/core_load_1.lpy` file for this test? I don't love the idea of having a seemingly unused file in the file tree.

Reply:
I personally don't like the idea of generating temporary files inside the test suite, but updated it anyway. 

It is common to have a test input vector checked in as parts of a test suite, though this particular one has to reside at a particular place in the test filesystem hierarchy rather than somewhere at the top.

Reply:
Generating test files is quite common and it is done throughout the test suite.

`basilisp.test` doesn't have as helpful of fixtures as does PyTest for the same, however.

Reply:
I think the main difference here is that the test files are generated in temp directories, while in this particular instance we are generating something inside the file hierarchy of the test suit itself.

Nevertheless thanks for the review